assets/js/components/ExecutionResults/checksUtils.test.js

```diff
@@ -179,5 +180,29 @@ describe('checksUtils', () => {
 
       expect(health).toBe('warning');
     });
+
+    it('should return true or false if a check is premium or not', () => {
+      const checkIds = [
+        faker.datatype.uuid(),
+        faker.datatype.uuid(),
+        faker.datatype.uuid(),
+        faker.datatype.uuid(),
+      ];
+      const expectedPremiumValues = [
+        true,
+        false,
+        undefined,
```

Comment:
Not a biggie: I would have sticked to the actual values that premium can have true/false.

Reply:
My idea was just to check some extra cases :D
No strong opinion here, i can change it if you want.

Reply:
It's just that those extra cases very likely won't happen. Up to you, you already got my approval :heavy_check_mark: :wink:

Reply:
+1 on this. `premium` is a boolean. I don't think we need to tests things that won't happen.